mne/viz/topo.py

```diff
@@ -330,8 +330,7 @@ def _plot_topo(
 
     for ax, ch_idx in my_topo_plot:
         if layout.kind == "Vectorview-all" and ylim is not None:
-            this_type = {"mag": 0, "grad": 1}[channel_type(info, ch_idx)]
-            ylim_ = [v[this_type] if _check_vlim(v) else v for v in ylim]
+            ylim_ = ylim.get(channel_type(info, ch_idx), None)
```

Comment:
```suggestion
            ylim_ = ylim.get(channel_type(info, ch_idx))
```
None is the default default value for `dict.get()`